docs/community/index.rst

```diff
@@ -30,7 +30,7 @@ Please join us on:
 Open Source Community Meeting 
 -----------------------------
 
-When: every other Tuesday, 9:00 AM Pacific Time.
+When: Every other Thursday, 11:00 AM Pacific Time.
```

Comment:
@davidmirror-ops is this accurate? And is the event link below (https://www.addevent.com/event/EA7823958) still correct?